docscan/session/create/filter/constants.go

```diff
@@ -0,0 +1,19 @@
+package filter
+
+type documentType string
+
+const identity documentType = "ID_DOCUMENT"
+
+// type restrictions string
```

Comment:
Is this supposed to be removed?

Reply:
They are constants for Document Restrictions, which I have started adding, but don't make use of these constants yet. Will uncomment and use for the next merge into Doc-Scan